### ISSUE TITLE
test_same_inspected_objects fails on ruby-trunk

### ISSUE THREAD

**Issue by eregon** [test-unit/test-unit#165]
See https://github.com/ruby/ruby/pull/2503#issuecomment-536274382
```
Failure: test_same_inspected_objects(Test::Unit::TestAssertions::TestAssertEqual::TestSystemMessage):
  Should have the correct message.
  <"<2019-09-29 11:36:45.499632827 +0200> expected but was\n<2019-09-29 11:36:45.499632827 +0200>."> expected but was
  <"<2019-09-29 11:36:45.499632827 +0200> expected but was\n<2019-09-29 11:36:45 +0200>.\n\ndiff:\n- 2019-09-29 11:36:45.499632827 +0200\n?                    ----------\n+ 2019-09-29 11:36:45 +0200">
```

Time#inspect now shows nanoseconds in its output on ruby-trunk.
This prevents running test-unit's tests in ruby/ruby's CI.

Could you fix the test to work with ruby-trunk?

**Comment by kou** [comment]
Thanks for your report.
I've released 3.3.4.

**Comment by eregon** [comment]
Thank you!
I made a PR to re-enable the tests in ruby/ruby: https://github.com/ruby/ruby/pull/2506
